archetype-common/src/main/java/org/apache/maven/archetype/creator/FilesetArchetypeCreator.java

```diff
@@ -1632,6 +1634,38 @@ private FileSet getUnpackagedFileSet(
         return createFileSet(excludes, false, filtered, group, includes, defaultEncoding);
     }
 
+    private String getFileCharsetEncoding(File detectedFile, String defaultEncoding) {
+        FileInputStream fileInputStream = null;
+        BufferedInputStream is = null;
+        try {
```

Comment:
try (InputStream in = new BufferedInputStream(new FileInputStream(...)))

Reply:
Okay, I now understand what an argument is, and I've updated my code.